packages/@lwc/template-compiler/src/__tests__/fixtures/parsing-errors/invalid-template-attribute/single-attribute/expected.js

```diff
@@ -1,7 +1,7 @@
 import { registerTemplate } from "lwc";
-const stc0 = [];
 function tmpl($api, $cmp, $slotset, $ctx) {
-  return stc0;
+  const { t: api_text } = $api;
+  return [api_text("Hello")];
```

Comment:
Could we also add a couple of Karma tests to make sure those templates are rendered properly? I would like to make sure the following cases are covered:


```html
<template>
  <!-- Single text node -->
  <template class="foo">
    Text
  </template>

  <!-- Single element -->
  <template class="foo">
    <span>Span</span>
  </template>

  <!-- Multiple elements -->
  <template class="foo">
    <span>Span #1</span>
    <span>Span #2</span>
  </template>

  <!-- Mixed -->
  <template class="foo">
    Before
    <span>Span #1</span>
    Middle
    <span>Span #2</span>
    After
  </template>
</template> 
```

Reply:
What about template-inside-template? Did this work before?

```html
<template class="foo">
  <template class="foo">
    text
  </template>
</template>
```

Or even:

```html
<template class="foo">
  before
  <template class="foo">
    middle
  </template>
  after
</template>
```